src/main/g8/backend/src/main/scala/$package$/backend/Launcher.scala

```diff
@@ -12,8 +13,15 @@ object Launcher extends CrossLogging {
   def main(args: Array[String]): Unit = {
     val startTime = System.nanoTime
 
-    val ctx = SpringContext.createApplicationContext("beans.conf")
-    val server = ctx.getBean(classOf[ApplicationServer])
+    val config: Config = ConfigFactory.load()
+    val rpcClientsService: RpcClientsService = new RpcClientsService(RpcClientsService.defaultSendToClientFactory)
+    val authService: AuthService = new AuthService(config.getStringList("auth.users"))
+    val chatService: ChatService = new ChatService(rpcClientsService)
+    val server = new ApplicationServer(
+      config.getInt("server.port"),
+      config.getString("server.statics"),
+      new DomainServices()(authService, chatService, rpcClientsService)
```

Comment:
I think implicit wiring in this case is a pretty good fit. These services are singletons anyway.